python/py-canmatrix/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        ebroecker canmatrix 0.6
+
+name                py-canmatrix
+categories-append   devel
+platforms           darwin
+license             BSD
+maintainers         {@ierofant gmail.com:aw.kornilov} openmaintainer
+description         Converting Can (Controller Area Network) Database Formats .arxml .dbc .dbf .kcd
+
+long_description    Canmatrix implements a "Python Can Matrix Object" which describes the can-communication \
+                    and the needed objects (Boardunits, Frames, Signals, Values, ...) \
+                    Canmatrix also includes two Tools (canconvert and cancompare) for converting and comparing \
+                    CAN databases.
+
+python.versions     27 34 35 36 37
```

Comment:
since this is a new port and presumably nothing depends on it, there is no real need to add older Python versions. In this case I would remove ```34``` and ```35```

Reply:
Done